src/content-helper/common/base-wordpress-provider.tsx

```
@@ -0,0 +1,316 @@
+
+			// Extract author data from _embedded.
+			const author = post._embedded?.author ? post._embedded.author[ 0 ] : null;
+
+			// Extract categories and tags data from _embedded.
+			// The first element in the array is categories, the second is tags.
+			if ( post._embedded && post._embedded[ 'wp:term' ] ) {
+				[ categories, tags ] = post._embedded[ 'wp:term' ];
+			}
+
+			// Get the post thumbnail.
+			if ( post._embedded && post._embedded[ 'wp:featuredmedia' ] ) {
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)